src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```
Лотарея це не кулька, вона їх створює